### PR TITLE
Change transform style on Android to fix perf

### LINKED ISSUE #751: Inverted scroll super laggy on Android
I want to use a  flashlist to show a inverted list. But if I use inverted prop on android it works very slow.
Default Flashlist: drops 5-10 FPS in dev mode (UI 60FPS)
Inverted Flashlist: drops 30-40 FPS in dev mode (UI also 20-40FPS)

```
const renderItem: ListRenderItem<IMessage> = ({ item, index }) => {
	return (
		<MessageItem id={item.id} message={item.message} date={item.date} />
	)};

 const keyExtractor = ({ id, date }: IMessage) => `${id}_${date}`;

 return (
	<SafeAreaView style={styles.container}>
		<FlashList
			ref={list}
			data={messages}
			keyExtractor={keyExtractor}
			renderItem={renderItem}
			contentContainerStyle={{
				paddingVertical: SCALE.XS,
			}}
			showsVerticalScrollIndicator={false}
                        estimatedItemSize={100}
		/>
	</SafeAreaView>
  );
```

## Current behavior

[inverted.webm](https://user-images.githubusercontent.com/99968085/216315189-47eef7bc-5d3b-4390-a06c-54ce1ad48b5a.webm)

## Expected behavior
UI 60 FPS

## To Reproduce

The flatlist also has the same behavior but after using [this](https://github.com/facebook/react-native/issues/30034#issuecomment-1411922104) patch UI stable 60fps but not working with FlashList
## Platform:

- [ ] iOS
- [+] Android

## Environment

1.4.1

### PR DESCRIPTION
**Pull request by naqvitalha**
## Description

resolves #751 

Change transform style to fix inverted performance issue on Android.

## Reviewers’ hat-rack :tophat:

Messages sample should perform flawlessly.

## Checklist

- [x] I have added a changelog entry following the [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) guidelines.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "up": "bundle install && yarn fixture-up && yarn e2e-up && yarn build && yarn fixture-web-up",
+    "up": "bundle install && yarn fixture-up && yarn e2e-up && yarn build",
```
Comment (naqvitalha) [inline package.json:31]: Removed due to some build issue.

File: src/native/config/PlatformHelper.android.ts [hunk 2]
```diff
@@ -2,6 +2,7 @@ import { BaseItemAnimator } from "recyclerlistview";
 
 const PlatformConfig = {
   defaultDrawDistance: 250,
+  invertedTransformStyle: { transform: [{ rotate: "180deg" }] },
```
Comment (fortmarek) [inline src/native/config/PlatformHelper.android.ts:5]: Maybe we can add a comment for why we're doing this and link to the original issues. Might be surprising to see this down the road.

**Review by fortmarek — APPROVED** [review]

**Comment by irisjae** [comment]
@KYZNCODE-Sebastian-Roese Indeed, checking the upstream patch suggests that we should pass `isInvertedVirtualizedList` to the inner ScrollView when using inverted lists on Android. This can be achieved by adding `overrideProps={{ isInvertedVirtualizedList: true }}` to the FlashList.

**Comment by saif-o99** [comment]
@naqvitalha Hey, Could you please tell which one is better in performance wish, `transform or scale`?
because the current solution creates an issue  for the scroll indicator and i've seen somewhere the solution is `scale` but i don't want to lose any performance.
Thanks.

**Comment by saif-o99** [comment]
@irisjae that works!. Thank you, any idea what does it do internally? is it just styling or may it affect the performance?

**Comment by irisjae** [comment]
Read my previous comment, and the linked commit. tldr, React Native fixed the issue, and this Flash List patch for the issue doesn't uses half of the upstream fix without the other half, which is the prop which I just pointed you to.

> @irisjae that works!. Thank you, any idea what does it do internally? is it just styling or may it affect the performance?

**Pull request merged**